src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx

```diff
@@ -149,65 +156,75 @@ const CardUserInfo: React.FC<YourScoreProps> = ({ hasRegistered, account, profil
         {subHeadingText}
       </Text>
       {shouldShowUserRanks && (
-        <RanksWrapper>
-          <Flex>
-            {volume > 0 ? (
+        <>
+          {profile.nft && volume > 0 && (
+            <Button mt="12px" variant="secondary" scale="sm" onClick={onPresentShareModal}>
+              {TranslateString(999, 'Share Score')}
+            </Button>
+          )}
+          <RanksWrapper>
+            <Flex>
+              {volume > 0 && (
+                <UserRankBox
+                  flex="1"
+                  title={TranslateString(1222, 'Rank in team').toUpperCase()}
+                  footer={`${TranslateString(
+                    999,
+                    `#${userLeaderboardInformation && global.toLocaleString()} Overall`,
+                  )}`}
+                  mr="8px"
+                >
+                  {!userLeaderboardInformation ? (
+                    <Skeleton height="26px" width="110px" />
+                  ) : (
+                    <TeamRankTextWrapper>
+                      <Heading textAlign="center" size="lg" mr="8px">
+                        #{team}
+                      </Heading>
+                      {medal.current}
+                    </TeamRankTextWrapper>
+                  )}
+                </UserRankBox>
+              )}
               <UserRankBox
                 flex="1"
-                title={TranslateString(1222, 'Rank in team').toUpperCase()}
-                footer={`${TranslateString(999, `#${userLeaderboardInformation && global.toLocaleString()} Overall`)}`}
-                mr="8px"
+                title={TranslateString(1224, 'Your volume').toUpperCase()}
+                footer={TranslateString(1226, 'Since start')}
+                mr={[0, null, '8px']}
```

Comment:
The diff is a bit confused by the code changes early in this file nudging a few things around

```js
 {profile.nft && volume > 0 && (
      <Button mt="12px" variant="secondary" scale="sm" onClick={onPresentShareModal}>
              {TranslateString(999, 'Share Score')}
      </Button>
)}
```

I have changed this dynamic prop, but it's not replacing the 'Rank in Team' mr as the diff suggests